src/i18n/injectIntlWithShim.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { injectIntl, intlShape } from 'react-intl';
-import LoggingService from '@edx/frontend-logging';
+import { logError } from '@edx/frontend-logging';
```

Comment:
Note: when i18n moves to its own library, we may want to configure logging on it as well, similar to what frontend-auth does.  It probably shouldn't assume that its consumer has logging initialized.